src/relation_engine_server/utils/arango_client.py

```diff
@@ -78,6 +78,16 @@ def init_collections():
         create_collection(coll_name, config)
 
 
+def init_views():
```

Comment:
I'd argue this function should live in `pull_spec.py`, as it's related to traversing the spec filetree and doesn't have any relation to arango